installation/scripts/testing.sh

```diff
@@ -40,6 +43,23 @@ do
     statusFailed=$(${kc} get cts ${suiteName}  -ojsonpath="{.status.conditions[?(@.type=='Failed')]}")
     statusError=$(${kc} get cts  ${suiteName} -ojsonpath="{.status.conditions[?(@.type=='Error')]}" )
 
+    succeededTestsNames=$(${kc} get cts ${suiteName} -ojsonpath="{.status.results[?(@.status=='Succeeded')].name}")
+    failedTestsNames=$(${kc} get cts ${suiteName} -ojsonpath="{.status.results[?(@.status=='Failed')].name}")
+
+    names=(${succeededTestsNames})
+    len="${#names[@]}"
+    if ((${succeededTests} < ${len})); then
```

Comment:
I think that it won't work if 2 tests finished at the same time. You will print only the name of the last one.